### PR TITLE
Expose lock ids

### PR DESCRIPTION
**Pull request by iorveth**
Part of https://github.com/Joystream/joystream/issues/2395

### CODE REVIEW

**Review by shamil-gadelshin — CHANGES_REQUESTED** [review]
Please, fix the compilation issue and comments of constants.

File: runtime-modules/council/src/lib.rs [hunk 1]
```diff
@@ -479,19 +493,31 @@ decl_module! {
...
         /// Interval for automatic reward payments.
         const ElectedMemberRewardPeriod: T::BlockNumber = T::ElectedMemberRewardPeriod::get();
+
         /// Interval between automatic budget refills.
         const BudgetRefillPeriod: T::BlockNumber = T::BudgetRefillPeriod::get();
 
+        /// Exports const - candidacy lock id.
+        const CandidacyLockId: LockIdentifier = T::CandidacyLock::lock_id();
+
+        /// Exports const - candidacy lock id.
```
Comment (shamil-gadelshin) [inline runtime-modules/council/src/lib.rs]: Incorrect comment.

Comment (iorveth) [reply runtime-modules/council/src/lib.rs]: Fixed comment and tests compilation here https://github.com/Joystream/joystream/pull/2536/commits/88248bef2ebe6d41fc2559ad7d06d87ec748d85c

**Review by shamil-gadelshin — APPROVED** [review]
LGTM

**Pull request merged**